### PR TITLE
Allows duplication of a redirect rule.

### PR DESCRIPTION
**Pull request by umanghome**
If I want a new Redirect rule that differs only a little bit from an existing one, it's a lot of effort to do so. It would be better if I can just duplicate an existing rule and then update it.

### CODE REVIEW

**Comment by ZIEXED** [comment]
Please move the duplicated rule below it's origin (I have a long list of roles, trying to duplicate my first rule resulted in it being placed in the most bottom and I didn't even noticed it)
also the button need more space

**Comment by umanghome** [comment]
Updated, please recheck.

File: css/redirector.css [hunk 1]
```diff
@@ -81,6 +81,10 @@ input[type="radio"] {
   text-decoration: none;
 }
 
+.btn.padded {
+  padding-left: 12px;
```
Comment (ZIEXED) [inline css/redirector.css]: remove padding-left

**Comment by einaregilsson** [comment]
Thanks for the PR. Will try to look at this later this week and merge it in if it's ok.

File: css/redirector.css (hunk 1 above)
Comment (umanghome) [reply css/redirector.css]: Why?

**Pull request merged**